src/linear_algebra/special_linear_group.lean

```diff
@@ -144,6 +158,23 @@ def to_GL : special_linear_group n R →* general_linear_group R (n → R) :=
 
 lemma coe_to_GL (A : special_linear_group n R) : ↑A.to_GL = A.to_lin'.to_linear_map := rfl
 
+section has_neg
+
+variables [fact (even (fintype.card n))]
+
+/-- Formal operation of negation on special linear group on even cardinality `n` given by negating
+each element. -/
+instance : has_neg (special_linear_group n R) :=
+⟨λ g,
+  ⟨- g, by simpa [nat.neg_one_pow_of_even (fact.out (even (fintype.card n))), g.det_coe] using
+  @det_smul _ _ _ _ _ ↑ₘg (-1)⟩⟩
```

Comment:
This line is telling you that we used `{}` on two arguments to `det_smul` when we should have used `()`. Would you mind adjusting the binders on that lemma so that you don't need an @ here?